lib/SiteConfig.py

```diff
@@ -284,11 +283,7 @@ def manage_session_vars(self, session_vars, write=False):
                 yaml.dump(self.session_vars, fhandle)
 
         with open(self.sv_path, "w") as fhandle:
-            if not self.printed_sv_msg:
-                install_logger.info("Dumping rendered site variables to {}".format(self.sv_path))
-                self.printed_sv_msg = True
-            else:
-                install_logger.debug("Dumping rendered site variables to {}".format(self.sv_path))
+            install_logger.debug("Dumping rendered site variables to {}".format(self.sv_path))
```

Comment:
How about we try to add some context to this message, like "Saving rendered site variables to be used with sat bootprep run --vars-file, to {}"